yargs.js

```diff
@@ -173,41 +174,49 @@ function Yargs (processArgs, cwd, parentRequire) {
   }
 
   self.boolean = function (keys) {
+    argsert('<array|string>', [].slice.call(arguments))
```

Comment:
Ah, no, I see that you're doing the same thing in argsert itself, so this is thankfully unnecessary.